src/DataFrame/Operations/Statistics.hs

```diff
@@ -55,9 +60,10 @@ frequencies name df = case getColumn name df of
           Just Refl -> T.pack c'
           Nothing -> (T.pack . show) c'
       initDf = empty & insertVector "Statistic" (V.fromList ["Count" :: T.Text,  "Percentage (%)"])
-    in L.foldl' (\df (col, k) -> insertVector (vText col) (V.fromList [k, k * 100 `div` total]) df) initDf counts
+    in L.foldl' (\df (col, k) -> insertVector (vText col) (V.fromList [toRowValue k, toRowValue $ k * 100.0  / total]) df) initDf counts
```

Comment:
For percentage we can do the fromIntegral just before division rather than having the counts be doubles. 

We can also format the string to round it for display:

```
import Text.Printf (printf)

printf "%.2f\n" (3.14159 :: Double)
```

Reply:
I went ahead with the second option, and here is the result:
```
--------------------------------------------------------------------------------
index |   Statistic    | <1H OCEAN |  INLAND  |  ISLAND  | NEAR BAY | NEAR OCEAN
------|----------------|-----------|----------|----------|----------|-----------
 Int  |      Text      | RowValue  | RowValue | RowValue | RowValue |  RowValue
------|----------------|-----------|----------|----------|----------|-----------
0     | Count          | 9136      | 6551     | 5        | 2290     | 2658
1     | Percentage (%) | 44.26     | 31.74    | 2.0e-2   | 11.09    | 12.88
```

Reply:
This looks good! Thank you.